### ISSUE TITLE
HTML5: AAC instead of MP3

### ISSUE THREAD

**Issue by Nallebeorn** [openfl/openfl#1069]
OpenFl decided to drop mp3 support for native targets because of the ridiculous fees required to distribute mp3-files. Yet, for the HTML5 target, the only option for audio in Internet Explorer is mp3 (in Safari as well, if you don't count wav, which is uncompressed)
IE, and Safari, also support AAC audio (m4a), which doesn't have the same legal issues... So it would be nice if that was an option in OpenFl HTML5.
This is actually really easy to change – in Sound.hx, it's just a matter of adding "m4a" to the SoundJS alternateExtensions array. The problem is that this will essentially break mp3 support, and thus probably some existing projects, because SoundJS will always try to load an m4a file instead if the browser supports the format, regardless of whether such a file actually exists...

Personally I'd say that's not a big problem – batch converting the mp3s to m4a should be easy (and many projects depending on mp3 support probably haven't actually paid the fees, anyway), or if that's not possible, anyone who wants so can easily change it back for themselves.
But that's why I didn't just make a PR immediately, I thought it would be a good idea to discuss it first, in case there's something more that should be done to ensure compatibility (I'm not actually quite sure of how strict you are with such things)

**Comment by KnightMearh** [comment]
I had tried exactly this before and it seemed successful: 

`alternateExtensions = [ "ogg", "m4a", "mp3", "wav" ];`

While OpenFL doesn't do MP3 directly, it still loads and plays just fine via SoundJS / browser.

I tried to commit the array update change to the github, but it merged with another change I was doing so was rejected. I just haven't gotten around in the past few weeks to separate it as I recently changed jobs/computers. If you want to push this minor change to github yourself, feel free.
